```
➜  etcd git:(pr-21618)  go test ./server/etcdserver/txn -run TestCheckRange -count=1
# go.etcd.io/etcd/server/v3/etcdserver/txn [go.etcd.io/etcd/server/v3/etcdserver/txn.test]
server/etcdserver/txn/range_bench_test.go:100:47: not enough arguments in call to Range
        have (context.Context, *zap.Logger, *mvcc.store, *etcdserverpb.RangeRequest)
        want (context.Context, *zap.Logger, mvcc.KV, *etcdserverpb.RangeRequest, bool)
FAIL    go.etcd.io/etcd/server/v3/etcdserver/txn [build failed]
FAIL
```
cc @Jefftree

--- a/server/etcdserver/txn/range_bench_test.go
+++ b/server/etcdserver/txn/range_bench_test.go
@@ -97,7 +97,7 @@ func BenchmarkRange(b *testing.B) {
 				b.ReportAllocs()
 				b.ResetTimer()
 				for b.Loop() {
-					_, _, err := Range(ctx, zap.NewNop(), s, req)
+					_, _, err := Range(ctx, zap.NewNop(), s, req, false)
 					if err != nil {
 						b.Fatal(err)
 					}

